pkg/server/server.go

```diff
@@ -0,0 +1,59 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	MetricsEndpointFlag = "metrics"
+	HealthzEndpointFlag = "healthz"
+	ListenEndpointFlag  = "listen-addr"
+)
+
+// Server represents a http server
+type Server struct {
+	mux        *http.ServeMux
+	listenAddr string
+	debug      bool
+}
+
+// New creates a new server
+func New(listenAddr string, debug bool) *Server {
+	return &Server{
+		mux:        http.NewServeMux(),
+		listenAddr: listenAddr,
+		debug:      debug,
+	}
+}
+
+// EnableMetricsEndpoint enables metrics endpoint
+func (s *Server) EnableMetricsEndpoint() {
+	s.mux.Handle("/metrics", promhttp.Handler())
+}
+
+// EnableHealthzEndpoint enables healthz endpoint
+func (s *Server) EnableHealthzEndpoint() {
+	s.mux.HandleFunc("/healthz", func(w http.ResponseWriter, req *http.Request) {
+		fmt.Fprintf(w, "OK")
```

Comment:
Should there a be a new line here?
```suggestion
		fmt.Fprintf(w, "OK\n")
```

Reply:
I would prefer not, as this is an http response.